unlock-app/src/__tests__/components/creator/CreatorLock.test.js

```diff
@@ -10,6 +11,25 @@ jest.mock('next/link', () => {
   return ({ children }) => children
 })
 
+/**
+ * Mocking web3Service
+ * Default objects yielded by promises
+ */
+class MockWebService extends EventEmitter {
+  constructor() {
+    super()
+    this.ready = true
+  }
+}
+
+let mockWeb3Service = new MockWebService()
```

Comment:
You can use just a regular object? Also I think we should actually not "integrate" this test so deeply. (see comments below)

```suggestion
let mockWeb3Service = {} 
```